ajax/libs/sinon.js/package.json

```diff
@@ -16,6 +16,16 @@
     "type": "git",
     "url": "https://github.com/sinonjs/sinon"
   },
+  "npmName": "sinon",
+  "npmFileMap": [
+    {
+      "basePath": "pkg",
+      "files": [
+        "sinon.js",
+        "sinon-no-sourcemaps.js"
```

Comment:
Hey sufuf3, I made the changes and squashed to one commit. Please let me know if there is anything else.